lang/user/id.xml

```diff
@@ -790,25 +794,26 @@ Setelah pengunduhan selesai, Vita3K akan restart secara otomatis dan kemudian me
 		<new_features>Baru Fitur di Versi ini {}</new_features>
 		<authors>Penulis</authors>
 		<comments>Komentar</comments>
-		<update>Update</update>
+		<update>Perbarui</update>
 		<version>Versi {}</version>
 	</vita3k_update>
 
-	<welcome name="SelamatDatang di Vita3K">
-		<vita3k>Vita3K PlayStation Vita Emulator</vita3k>
+	<welcome name="Selamat datang di Vita3K">
+		<vita3k>Emulator Vita3K PlayStation Vita</vita3k>
 		<about_vita3k>Vita3K adalah emulator PlayStation Vita sumber terbuka yang ditulis dalam C++ untuk Windows, Linux, macOS dan Android.</about_vita3k>
 		<development_stage>Emulator masih dalam tahap awal sehingga umpan balik dan pengujian apa pun sangat dihargai.</development_stage>
 		<about_firmware>Untuk memulai, silakan instal paket firmware dan font PS Vita.</about_firmware>
 		<vita3k_quickstart>Panduan lengkap tentang cara menyiapkan Vita3K dapat ditemukan di</vita3k_quickstart>
-		<quickstart>MulaiCepat</quickstart>
+		<quickstart>Mulai Cepat</quickstart>
 		<page>halaman.</page>
-		<check_compatibility>Konsultasikan game Komersial dan daftar kompatibilitas Homebrew untuk melihat apa yang sedang berjalan.</check_compatibility>
+		<check_compatibility>Konsultasikan game Komersial dan daftar kompatibilitas Homebrew untuk melihat apa yang sedang
```

Comment:
For this one it can be shortened further, to

```xml
<check_compatibility>Cek kompatibilitas game Komersial dan Homebrew</check_compatibility>
```
The words after homebrew are unnecessary as this should be descriptive enough.

Reply:
> For this one it can be shortened further, to
> 
> ```
> <check_compatibility>Cek kompatibilitas game Komersial dan Homebrew</check_compatibility>
> ```
> 
> The words after homebrew are unnecessary as this should be descriptive enough.

But this is sentence that corresponds to English.

Reply:
> > For this one it can be shortened further, to
> > ```
> > <check_compatibility>Cek kompatibilitas game Komersial dan Homebrew</check_compatibility>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > The words after homebrew are unnecessary as this should be descriptive enough.
> 
> But this is sentence that corresponds to English.

Yes, but omitting it does not reduce its meaning in our language. You can keep either of them if unsure as both of them are the same.